cmake/modules/FindBoost.cmake

```diff
@@ -780,10 +779,10 @@ function(_Boost_GUESS_COMPILER_PREFIX _ret)
           set(_boost_COMPILER "-gcc${_boost_COMPILER_VERSION}")
         endif()
       endif()
-    elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-      # TODO: Find out any Boost version constraints vs clang support.
-      set(_boost_COMPILER "-clang${_boost_COMPILER_VERSION}")
     endif()
+  elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```

Comment:
could you move this change to a separated commit, and just use the upstream version of https://github.com/Kitware/CMake/blob/master/Modules/FindBoost.cmake .

Reply:
Makes sense. I was checking why it wasn't using the upstream module already and saw your commit: 027164e156ee9bf95804ea8b51bbfbd9592fe8c1. Is cmake 3.5.1 still a concern? Is there a list of supported distros?

Reply:
@petrutlucian94 the minimum cmake version we require is `3.10.2`. see also bbb69fe793948d0791e29f547e18f8fa996c32ea

Reply:
Right, haven't noticed that. Thanks!